initial german translations

rschnapka opened the pull request.

rschnapka committed:
initial german translations

pedrobaeza commented on auth_admin_passkey/i18n/de.po:
```diff
@@ -0,0 +1,123 @@
+#: code:addons/auth_admin_passkey/model/res_users.py:66
+#, python-format
+msgid "<pre>User with login '%s' has the same password as you.</pre>"
+msgstr ""
```
Why this line feed?

pedrobaeza replied:
It is still there.

pedrobaeza commented on cron_run_manually/i18n/de.po:
```diff
@@ -0,0 +1,51 @@
+#: model:_description:0
+#: model:ir.model,name:cron_run_manually.model_ir_cron
+msgid "ir.cron"
+msgstr ""
```
Translate this, although it is also `ir.cron`

rschnapka committed:
removed unneeded linefeeds

rschnapka committed:
translation ir.cron added

rschnapka committed:
remove more line feeds

rschnapka committed:
remove extra line feed manually

Program Localize doesn't seem to remove the extra LF on single-lines? Maybe I should try again to get poedit running on my ubuntu.

pedrobaeza commented:
Strange the problem you're having with Lokalize...

:+1:

rschnapka commented:
maybe it's just me :-) as long as it gets fixed...

yvaucher commented:
:+1:

rschnapka commented:
1. Is this runbot-problem still due to my not approved license agreement?
2. po-editor problem ist not restricted to 'Lokalize' - 'PO Edit' also breaks the lines with excess linefeeds when they are longer than ~80 chars...

pedrobaeza commented:
No, runbot it's in beta state, and thus it breaks frequently.

I see the problem with localization tools. I always edit in raw mode, but I admit that they are more confortable. I remember that in past, these line feeds are parsed and introduces in texts (because the PO parser is a custom one). That's why I tell you to change, but you can try to see if this behaviour remains.
